### PR TITLE
Raven db 7276

### PR DESCRIPTION
**Pull request by arekpalinski**

### CODE REVIEW

File: src/Raven.Client/Server/ETL/EtlConfiguration.cs [hunk 1]
```diff
@@ -29,5 +32,10 @@ public virtual bool Validate(out List<string> errors)
         }
 
         public EtlType EtlType => Destination is RavenDestination ? EtlType.Raven : EtlType.Sql;
+
+        public ulong GetTaskKey()
+        {
+            return _taskKey ?? (_taskKey = Hashing.XXHash64.Calculate(Destination.Name, Encodings.Utf8)).Value;
```
Comment (ayende) [inline src/Raven.Client/Server/ETL/EtlConfiguration.cs:38]: I'm worried about case sensitivity here, should we do something here?

File: src/Raven.Server/Documents/DatabasesLandlord.cs [hunk 2]
```diff
@@ -151,6 +151,32 @@ public void ClusterOnDatabaseChanged(object sender, (string dbName, long index,
             }
         }
 
+        public void ClusterOnDatabaseValueChanged(object sender, (string dbName, long index, string type) t)
+        {
+            if (DatabasesCache.TryGetValue(t.dbName, out var task) == false)
+            {
+                // if the database isn't loaded, but it is relevant for this node, we need to create
```
Comment (ayende) [inline src/Raven.Server/Documents/DatabasesLandlord.cs:158]: How do we know here that it is relevant for this node?

File: src/Raven.Server/Documents/DocumentDatabase.cs [hunk 3]
```diff
@@ -571,11 +571,31 @@ public void IncrementalBackupTo(string backupPath)
...
+            try
+            {
+                if (_databaseShutdown.IsCancellationRequested)
+                    ThrowDatabaseShutdown();
+
+                SubscriptionStorage?.HandleDatabaseValueChange();
+            }
+            catch
+            {
+                if (_databaseShutdown.IsCancellationRequested)
```
Comment (ayende) [inline src/Raven.Server/Documents/DocumentDatabase.cs:590]: Why do we have this here? It swallow all exceptions _except_ shutdown?

File: src/Raven.Server/Documents/ETL/EtlProcess.cs [hunk 4]
```diff
@@ -409,7 +412,24 @@ public void Run()
...
-                        _storage.StoreLastProcessedEtag(Destination, Name, Statistics.LastProcessedEtag);
+                        var command = new StoreEtlStatusCommand(new EtlProcessStatus
+                        {
+                            Destination = Destination.Name,
+                            TransformationName = Name,
+                            LastProcessedEtag = Statistics.LastProcessedEtag
+                        }, Database.Name);
+
+
+                        var (etag, _) = _serverStore.SendToLeaderAsync(command).Result;
```
Comment (ayende) [inline src/Raven.Server/Documents/ETL/EtlProcess.cs:423]: shouldn't we also do a `Wait(CancellationToken)` here?

File: src/Raven.Client/Server/ETL/EtlConfiguration.cs (hunk 1 above)
Comment (arekpalinski) [reply src/Raven.Client/Server/ETL/EtlConfiguration.cs:38]: Right, I'll add `ToLowerInvariant` here

File: src/Raven.Server/Documents/DatabasesLandlord.cs (hunk 2 above)
Comment (arekpalinski) [reply src/Raven.Server/Documents/DatabasesLandlord.cs:158]: Sorry, I forgot to check against the db record (`record.Topology.RelevantFor(_serverStore.NodeTag)`

File: src/Raven.Server/Documents/DocumentDatabase.cs (hunk 3 above)
Comment (arekpalinski) [reply src/Raven.Server/Documents/DocumentDatabase.cs:590]: Just blindly copied it from `StateChanged` to be frank. Shouldn't it be like that?

File: src/Raven.Server/Documents/ETL/EtlProcess.cs (hunk 4 above)
Comment (arekpalinski) [reply src/Raven.Server/Documents/ETL/EtlProcess.cs:423]: Yes, it should. I'll add it

File: src/Raven.Server/Documents/DocumentDatabase.cs (hunk 3 above)
Comment (ayende) [reply src/Raven.Server/Documents/DocumentDatabase.cs:590]: Not sure, to be honest. Need to check upward to see what is the behavior. 
Swallowing the exception is probably not okay anyway

**Pull request merged**